examples/using-cron-jobs/configs/.env

```diff
@@ -1,3 +1,5 @@
 APP_NAME=sample-cron
 
-TRACE_EXPORTER=zipkin
\ No newline at end of file
+TRACE_EXPORTER=zipkin
+TRACER_HOST=127.0.0.1
+TRACER_PORT=9411
```

Comment:
why are we adding host and port instead of url?

Reply:
the former only add a host without a port , so i add it  or we can replace to url

Reply:
yes, we can add URL instead.. since it is new format